wbooks-api/app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class BookSerializer < ActiveModel::Serializer
+  attributes :id, :title, :author, :genre, :publisher, :year
+
+  has_many :rents, key: :actual_rent do
+    object.rents.where('restitution >= ?', Date.current).first
```

Comment:
what if you have an actual rent and a rent for the future? you should check that the current date is between the loan and the restitution dates

Reply:
also, when creating a rent, you should check that they don't overlap

Reply:
Ok, I'll make the changes!